### PR TITLE
New fire alarm sound

### PR DESCRIPTION
**Pull request by nero1024**
Alternate Title: "Made the fire alarm sound more annoying"

:cl: Nero1024
soundadd: New intergalactic fire codes have mandated that all fire alarms on commercial space stations be made to sound more distinct (and ear-piercing).
/:cl:

The space station is a high-occupancy facility right? Shouldn't the fire alarms sound like they're for such places?

### CODE REVIEW

File: code/game/machinery/firealarm.dm [hunk 1]
```diff
@@ -124,7 +124,7 @@
 	last_alarm = world.time
 	var/area/A = get_area(src)
 	A.firealert(src)
-	playsound(src.loc, 'goon/sound/machinery/FireAlarm.ogg', 75)
+	playsound(src.loc, 'sound/machines/firealarm.ogg', 75)
```
Comment (ninjanomnom) [inline code/game/machinery/firealarm.dm]: Remove the old sound from the goon folder and change over the only other use of this sound as well

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Comment by Jordie0608** [comment]
It really is more annoying.
What's the source on this?

File: code/game/machinery/firealarm.dm [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#define FIREALARM_COOLDOWN 67 // Chosen fairly arbitrarily, it is the length of the audio in FireAlarm.ogg. The actual track length is 7 seconds 8ms but but the audio stops at 6s 700ms
+#define FIREALARM_COOLDOWN 38 // Chosen fairly arbitrarily, it is the length of the audio in FireAlarm.ogg. The actual track length is 7 seconds 8ms but but the audio stops at 6s 700ms
```
Comment (Jordie0608) [inline code/game/machinery/firealarm.dm]: Update the comment if you're changing this.

**Comment by nero1024** [comment]
@Jordie0608 The fire alarm is sampled from a audio recording I made. Said sample was a very short >50ms loop and involved some synthesis shenanigans involving OpenMPT. The silence after the sound is intentional, mostly to make sure the sample loops like it's supposed to (~1sec of silence after 3 beeps), the silence shouldn't affect the filesize much.

**Comment by Jordie0608** [comment]
Ok that's fine, I just assumed otherwise that it wasn't intended. The files are only 15KB different so no worry there.

**Comment by nfreader** [comment]
This is a bad PR. Using a sound effect from a real fire alarm will cause alarm fatigue, and could result in someone mistaking an actual fire alarm for just another false alarm in SS13. Or vice versa. There’s a reason you don’t hear this alarm in many other games.

**Comment by ShizCalev** [comment]
itt; how to make firealarms even more annoying

**Comment by nero1024** [comment]
One thing I can do to make it even more annoying is amplify the sample by 2000% and set the playsound volume arg to something stupidly high like 32767.

**Comment by Anonymous583** [comment]
Don't do this.

**Comment by Qustinnus** [comment]
bad sound

**Comment by Arianya** [comment]
Didn't we go through this a while back with someone making the fire alarm really shrill and repeat really often and when it got merged it caused endless bitching until it was changed to its present state

**Comment by ShizCalev** [comment]
What @nfreader said above. This borders on a personal safety issue since it is a sound from an ACTUAL fire alarm.

**Comment by Iamgoofball** [comment]
normally i would be all for this but alarm fatigue is a real thing

**Pull request closed without merging**